cypher/cypher-docs/src/docs/dev/ql/administration/security/show-privileges-syntax.asciidoc

```diff
@@ -1,27 +0,0 @@
-.Show privileges command syntax
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/manage-privileges.adoc